examples/lib/stories/components/priority.dart

```diff
@@ -52,6 +52,6 @@ class Priority extends BaseGame with HasTapableComponents {
       Square(Vector2(170, 150)),
       Square(Vector2(110, 150)),
     ];
-    addAll(squares);
+    components.addAll(squares);
```

Comment:
addAll is one that was considering if could be added proxied

Reply:
Now when `addAll` is back that one should be used

Reply:
ah I added only to BaseGame. let me add to BaseComp